src/components/catalog/DisplayPage.tsx

```diff
@@ -335,7 +335,8 @@ const DisplayPage = ({
             {showPipelineButton && (
               <Button
                 style={{
-                  width: "45%",
+                  width: "55%",
```

Comment:
This is a large and dynamic component, so setting width based on some percentage of its parent's size could be a brittle solution.

I think it makes more sense here to have the minimum width be dependent on the size of its contents instead of the size of the parent.

Reply:
Alright @jennydaman width:fit-content would do it.. Thanks !! for the feedback .. I have pushed the changes.

Reply:
It always seems like CSS has these features that you gotta dig around for.